koku/api/report/ocp/query_handler.py

```diff
@@ -361,3 +364,19 @@ def add_current_month_deltas(self, query_data, query_sum):
         self.query_delta = {"value": total_delta, "percent": total_delta_percent}
 
         return query_data
+
+    def format_vm_csv_response(self, query_data):
```

Comment:
Nit Pick:
I think comment you left on 277 provides enough context to describe what is happening in these four lines of code. Therefore, I wouldn't add a new method here since this method will ever only be called once. We could just include these 4 lines in the correct place above instead of jumping around. Makes traceability easier in the future.